sys/include/senml.h

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ * Copyright (C) 2016 Lennart Dührsen <lennart.duehrsen@fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+
+/**
+ * @ingroup     senml
```

Comment:
Also, if you want to add it to an existing group its `@addtogroup`, `@ingroup` makes it a subgroup of `senml`. I think you want to use `@defgroup` here however and use `sys` as your `@ingroup`. Remember to also provide a human-readable name for the `senml` group. I propose `SenML`